docs/development-guidelines/operational-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# Operational guidelines
+
+The following guidelines are meant to help teams and contributors organize themselves and establish roles within template authoring flows to ensure consistency, quality, and collaboration.
+
+## Template owner(s)
+
+The template owner is the person who owns the template and acts as a point of contact to receive notification when applicable. Typically this role is filled by a core maintainer, who may or may not be code owner, but probably has some administrative persmissions in the overall repository.
+
+This code owner may the one organizing the work and reporting progress. 
+
+## Project (technical lead)
+
+A project technical lead may be the template owner, or a designated team member representing the technology, tool or service to high-light with the purpose of providing technical direction, coherence, and quality across the project. 
+
+A technical lead, depending on their seniority and competence, may be the ultimate decision maker and coordinator of all technical efforts, although when templates integrate multiple tools and service, we recommend to scope per technology and have experts on each category assuming this role.
+
+## Case scenarios and specification
+
+Before the team starts coding, make sure to have designed scenarios, use-cases and specification, answering thes questions
+
+- What services, tools and products are we showcasing?
```

Comment:
Nit: recommend sweeping through and applying the Oxford comma on the lists, like in "services, tools, and products"

Reply:
fixed!